src/types/Currency.sol

```diff
@@ -6,6 +6,7 @@ import {IERC20Minimal} from "../interfaces/external/IERC20Minimal.sol";
 type Currency is address;
 
 using {greaterThan as >, lessThan as <, greaterThanOrEqualTo as >=, equals as ==} for Currency global;
+using CurrencyLibrary for Currency global;
```

Comment:
`using CurrencyTeller for Currency global;`?

Reply:
I think this is referring to the other CurrencyLibrary defined below in this contract

Reply:
We are making a change to pull the CurrencySettleTake library out of core as its really only used in tests and integration contracts. So it should be clearer

Reply:
Gotcha. I was suggesting adding `using CurrencyTeller for Currency global;` besides `using CurrencyLibrary for Currency global;`.

Reply:
yeah would also make sense - we're going to remove that library and put it into periphery now. have reverted the renaming